lib/eventsource-polyfill.js

```diff
@@ -1,5 +1,6 @@
 var EventSource = require('./eventsource')
 if (typeof window === 'object') {
   window.EventSource = window.EventSource || EventSource
+} else if (typeof global === 'object') {
+  global.EventSource = global.EventSource || EventSource
```

Comment:
Is it better to create the window object in node, or attach to global instead?
eg `if (typeof window !== object) global.window = {}`

`new EventSource` vs `new window.EventSource`